lib/request.js

```diff
@@ -225,11 +225,21 @@ Object.defineProperties(Request.prototype, {
       }
     }
   },
-  hostname: {
+  host: {
     get () {
       return this.raw.headers.host || this.raw.headers[':authority']
     }
   },
+  hostname: {
+    get () {
+      return (this.host).split(':')[0]
+    }
+  },
+  port: {
```

Comment:
Small nit:

Why not get it from the `this.hostname`?

Reply:
Should we also consider the case where no port is defined within the URL? for the `host/:authority`?

Reply:
it would fail over here, at [Line 44](https://github.com/aarontravass/fastify/blob/4f28ce31f861658840ee2f4a822e8e933dae5acc/test/trust-proxy.test.js#L44). `host` header was set and port was taking it from the `req.host` but `req.host` was example.com, so it would break.

Reply:
> Should we also consider the case where no port is defined within the URL? for the `host/:authority`?

yeah, I was wondering the same. Should I just put a fall back to null?

Reply:
ping @metcoder95

Reply:
Sorry. Sure, fallbacking to `null` works 👍